source/PlanetPanel.cpp

```diff
@@ -367,6 +376,11 @@ void PlanetPanel::CheckWarningsAndTakeOff()
 			out << (outfitsToSell > 1 ? "s" : "");
 			out << " that none of your ships can hold.";
 		}
+		// Warn about unique items you sold.
+		else if(leftUniques)
+		{
+			out << "If you take off now you won't be able to re-purchase unique outfits you sold at the outfitter.";
```

Comment:
It should probably say how many unique outfits will be sold

Reply:
An alternative would be to list the outfits as per https://github.com/endless-sky/endless-sky/pull/9112#discussion_r1281320297, and tbh I prefer that solution.